README.md

```diff
@@ -1 +1,8 @@
-# process_yandex_tf_files
+# Process yandex terraform files
+
+This script moves user and database recources from cluster resource to separate ones.
+
+## Usage
+```
+python3 ./process_tf.py -s source_dir -d dest_dir
```

Comment:
I suggest to save file to the same directory but with suffix `-updated.tf` cause terraform initialised in the source_dir. Or even just update original files.